### PR TITLE
chore(package upgrades): Upgrade AWS SDK from v2 to v3

### PR DESCRIPTION
**Pull request by nina-py**
## Goal

Upgrade outdated/deprecated packages

- The S3 uploads to the bucket that holds thumbnails for curated corpus items were made using the old AWS SDK - event bridge code uses the new one.

The old packages are on the verge of going into maintenance mode and were polluting integration tests output with endless deprecation notices:

> (node:724) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

> Please migrate your code to use AWS SDK for JavaScript (v3).
> For more information, check the migration guide at https://a.co/7PzMCcy

Some code changes were required in addition to swapping out the packages.

## Todos

- [x] Test on Dev

### CODE REVIEW

File: src/test/helpers/createApprovedItemHelper.ts [hunk 1]
```diff
@@ -41,20 +41,20 @@ export async function createApprovedItemHelper(
   const random = Math.round(Math.random() * 1000);
 
   // randomize number of authors
-  const authorCount = faker.datatype.number({ min: 1, max: 3 });
+  const authorCount = faker.number.int({ min: 1, max: 3 });
```
Comment (nina-py) [inline src/test/helpers/createApprovedItemHelper.ts:44]: Fixed Faker 8.0 deprecation notices as they were flooding test output

**Comment by cmharlow** [comment]
Running integration tests locally fail. Guessing some local (docker-compose) setup is missing, but debugging.

File: src/admin/aws/upload.integration.ts [hunk 2]
```diff
@@ -8,14 +8,16 @@ import { ApprovedItemS3ImageUrl } from '../../shared/types';
 
 const testFilePath = __dirname + '/test-image.jpeg';
 
+// Port is not used in CircleCI, so make it optional in the regex.
+// Also, AWS Upload turns the test file in CircleCI into a PNG.
```
Comment (kschelonka) [inline src/admin/aws/upload.integration.ts:12]: that's strange

Comment (nina-py) [reply src/admin/aws/upload.integration.ts:12]: @kschelonka, yes - this is only happening in CircleCI, on Dev the uploaded image keeps its original format

**Review by kschelonka — APPROVED** [review]

**Comment by nina-py** [comment]
Thanks @cmharlow, those local test failures are also present on the `main` branch and are possibly (?) related to other package or Docker image updates - I'll add a chore to the backlog to remember to fix those one day.

**Pull request merged**